src/main/java/io/confluent/idesidecar/websocket/messages/MessageType.java

```diff
@@ -0,0 +1,13 @@
+package io.confluent.idesidecar.websocket.messages;
+
+
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+/**
+  * Pseudo enum holding websocket message types of concern to sidecar.
+  * Other message types may exist for use for IDE workspace <--> workspace communication.
+ **/
+@RegisterForReflection
+public class MessageType {
+  public static final String WORKSPACE_COUNT_CHANGED = "WORKSPACE_COUNT_CHANGED";
+}
```

Comment:
No, because we want to make it so that workspace -> other workspace broadcast messages won't need any sidecar-side PRs. For that to work, we need to model things on sidecar so that it's fine that it is ignorant of some / many / most eventual message types.